src/segmentmytif/features.py

```diff
@@ -89,16 +91,23 @@ def extract_flair_features(input_data: ndarray, model_scale=1.0) -> ndarray:
     return output[0, :, :, :]
 
 
-def load_model(model_scale:float):
+def load_model(model_scale:float, models_dir: Path = Path("models")):
```

Comment:
```python
DEFAULT_MODELS_DIR = Path(__file__).parent / "../../models"

...
def load_model(model_scale:float, models_dir: Path = DEFAULT_MODELS_DIR):
```

I would suggest to use the relativa path for the default model directory. Here I wrote an example where I implemented in #47.

Reply:
I agree. I deliberately didn't change it yet in this PR, because it's part of your PR already. Not sure if it matters, as we'll have a conflict anyway.